includes/Modules/Analytics.php

```diff
@@ -595,33 +563,44 @@ protected function get_datapoint_services() {
 	 * @return array Map of datapoints to their definitions.
 	 */
 	protected function get_datapoint_definitions() {
-		$map = parent::get_datapoint_definitions();
-
-		$map['POST:create-account-ticket'] = array_merge(
-			$map['POST:create-account-ticket'],
-			array(
+		return array(
+			'GET:accounts-properties-profiles' => array( 'service' => 'analytics' ),
```

Comment:
`GET:accounts-properties-profiles` should be after `POST:account-id`.

Reply:
Updated.